.github/workflows/check_translations.yml

```diff
@@ -0,0 +1,51 @@
+name: Check translation files
+
+on:
+  pull_request:
+    paths:
+      - 'java/code/src/com/redhat/rhn/frontend/strings/**'
+      - 'backend/**'
+      - 'client/rhel/yum-rhn-plugin/**'
+      - 'client/rhel/mgr-daemon/**'
+      - 'client/rhel/spacewalk-client-tools/**'
+      - 'web/**'
+      - 'susemanager/**'
+      - 'spacecmd/**'
+
+jobs:
+ run:
+   name: Locally update translation files
+   if: github.repository == 'uyuni-project/uyuni'
+   runs-on: ubuntu-latest
+   steps:
+   - name: Cancel Previous Runs
+     uses: styfle/cancel-workflow-action@0.8.0
+     with:
+         access_token: ${{ github.token }}
+
+   - name: Checkout repo
+     uses: actions/checkout@v2
+
+   - name: Setup tooling
+     run: sudo apt-get install -y make git gettext intltool python3
```

Comment:
I want to make sure those tools are installed. I can't be certain because some actions run in docker containers. This why I am keeping the list explicitly